pages/agent/clusters.md

```diff
@@ -0,0 +1,53 @@
+# Clusters
+
+Clusters are a new way of managing your Buildkite agents. They allow teams to self-manage their agent pools, let admins create isolated sets of agents and pipelines within the one Buildkite organization, and help to make agents and queues more discoverable across your organization.
+
+The following diagram shows the architecture difference after enabling clusters.
+
+<%= image "clusters-architecture.png", alt: "Diagram showing existing architecture and architecture with clusters" %>
+
+In the diagram, you'll see that clusters encapsulate groups of agents and pipelines. These groups enable the following behaviors:
+
+* Clusters are viewable to your entire organization, allowing engineers to understand better what agents and queues are available in their pipelines.
+* You can grant cluster-admin access to individual users or teams to let them self-administer their own sets of agents. Cluster admins can create and delete queues, access agent tokens, and add and remove pipelines.
+* Queues can be created within a cluster and validated when a pipeline's configuration is saved during pipeline upload, and when a new agent connects. This helps to make queues more discoverable to pipeline authors and helps to prevent ghost jobs that never run because they had a typo in their pipeline.yml's queue name.
+* Pipelines now belong to a cluster. Pipelines can only schedule jobs on the cluster it belongs to and can only trigger builds on pipelines in the same cluster.
```

Comment:
Can pipelines be unclustered? Should this be saying that you can add pipelines to clusters?

Reply:
Yes, pipelines can also be unclustered. So perhaps some light rephrasing here would help. Something like this? "Pipelines can belong to a cluster. Such pipelines can only schedule jobs..."